src/beamlime/frontend/zmq_client.py

```diff
@@ -0,0 +1,167 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
I'm guessing most of the Flask, ZMQ code was written by copilot?

I didn't look so much into the changes, as I understand close to none of it.
However, this is at an experimental stage, I don't see a reason not to merge this.